### PR TITLE
chore: add compressed-size-action

### PR DESCRIPTION
**Pull request by kangju2000**
# Overview

close https://github.com/toss/suspensive/issues/967

Add the 'compressed-size-action' to check the size of the dist folder in packages.

## PR Checklist

- [x] I did below actions if need

1. I read the [Contributing Guide](https://github.com/toss/suspensive/blob/main/CONTRIBUTING.md)
2. I added documents and tests.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -39,3 +39,17 @@ jobs:
...
+
+  size:
+    name: Size
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/pnpm-setup-node
+      - run: pnpm install --frozen-lockfile
+
+      - name: Report build size
```
[the remark this answers is not in the document]
Comment (manudeli) [reply .github/workflows/ci.yml:51]: There is solution to reduce time to do this https://github.com/marketplace/actions/compressed-size-action?version=2.6.0#customizing-the-build

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -39,3 +39,17 @@ jobs:
...
+        uses: preactjs/compressed-size-action@v2
+        with:
+          pattern: 'packages/**/dist/**'
```
Comment (manudeli) [inline .github/workflows/ci.yml:54]: prepack make only build packages

```suggestion
        with:
          build-script: "pnpm prepack"
          pattern: 'packages/**/dist/**'
```

Comment (manudeli) [reply .github/workflows/ci.yml:54]: Please commit suggestion if you agree this change

File: .github/workflows/ci.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kangju2000) [reply .github/workflows/ci.yml]: 👍👍👍

|as-is|to-be|
|---|---|
|image|image|

File: .github/workflows/compressed-size.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+env:
+  TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+  TURBO_TEAM: ${{ vars.TURBO_TEAM }}
+  CI: ${{ vars.CI }}
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```
Comment (kangju2000) [inline .github/workflows/compressed-size.yml:12]: @manudeli 
I'm not sure about this code. Is it necessary?"

Comment (manudeli) [reply .github/workflows/compressed-size.yml:12]: Yes! This concurrency keyword prevent unnecessary workflow run if new commit made during running worflow exist

**Review by manudeli — APPROVED** [review]
We can check how big size our library in Pull Request everytime we make change 🚀🚀🚀🚀🚀

**Comment by manudeli** [comment]
@kangju2000 could you make this action check only cjs, js, mjs please?

**Pull request merged**